File: eBiblioteka/eBiblioteka.Core/Entities/RecommendResult.cs

namespace eBiblioteka.Core
{
    public class RecommendResult:BaseEntity
    {
        public int BookId { get; set; }
        public int FirstCobookId { get; set; }
        public int SecondCobookId { get; set; }
        public int ThirdCobookId { get; set; }
    }
}

File: eBiblioteka/UI/mobile_ebiblioteka/lib/models/recommend_result.dart
import 'package:json_annotation/json_annotation.dart';

part 'recommend_result.g.dart';

@JsonSerializable()
class RecommendResult {
  int? id;
  int? bookId;
  int? firstCobookId;
  int? secondCobookId;
  int? thirdCobookId;

  RecommendResult(this.id, this.bookId, this.firstCobookId, this.secondCobookId, this.thirdCobookId);

  factory RecommendResult.fromJson(Map<String, dynamic> json) =>
      _$RecommendResultFromJson(json);

  Map<String, dynamic> toJson() => _$RecommendResultToJson(this);
}


